yarn.lock

```diff
@@ -6541,14 +6541,6 @@ eslint-module-utils@^2.8.0:
   dependencies:
     debug "^3.2.7"
 
-eslint-plugin-flowtype@^5.10.0:
-  version "5.10.0"
```

Comment:
Pas de modification dans le package.json, donc il ne devrait pas y avoir de modifications sur le yarn.lock

Reply:
I missed something in #889 this is the correct version of lock